src/Input/Input.js

```diff
@@ -323,9 +368,16 @@ module.exports = kind(
 	/**
 	* @private
 	*/
-	input: function () {
+	input: function (sender, e) {
 		var val = this.getNodeProperty('value');
 		this.setValue(val);
+		if(this._enabledValidity) {
+			if(!e.target.validity.valid) {
+				this.bubble('onenabledValidity');
```

Comment:
You can shorten this as single line.
this.bubble(validity ? 'onenabledValidity' : 'ondisabledValidity');